cli/command/service/opts.go

```diff
@@ -509,6 +548,9 @@ func addServiceFlags(flags *pflag.FlagSet, opts *serviceOptions) {
 
 	flags.StringVarP(&opts.workdir, flagWorkdir, "w", "", "Working directory inside the container")
 	flags.StringVarP(&opts.user, flagUser, "u", "", "Username or UID (format: <name|uid>[:<group|gid>])")
+	flags.Var(&opts.credentialSpec, flagCredentialSpec, "Credential spec for managed service account (Windows only)")
```

Comment:
I wonder if `(Windows only)` is necessary since the flag is already conditional on a Windows daemon. I don't think we say `(Linux only)` for Linux-specific flags.